src/Common/Core/Impl/Shell/IApplicationConstants.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core.Shell {
+    /// <summary>
+    /// Defines application constants such as locale, registry key names, etc.
+    /// Implemented by the host application. Imported via MEF.
+    /// </summary>
+    public interface IApplicationConstants {
```

Comment:
We already have `ApplicationWindowHandle` that I think can be moved into this interface (cause it won't change till the end of the process).

---
In reply to: [80990779](https://github.com/Microsoft/RTVS/pull/2408#discussion_r80990779) [](ancestors = 80990779)